lib/api/core/hotelClerk.js

```diff
@@ -790,3 +793,44 @@ function subscribeToRoom (roomId, requestObject, context) {
 
   return deferred.promise;
 }
+
+
+TokenizedConnectionsController = function(kuzzle) {
+  var
+    tokenizedConnections = {},
+    runTimer,
+    timer = null;
+
+  this.add = function(token, context) {
+    tokenizedConnections[token._id] = {
+      expiresAt: token.expiresAt,
+      connection: context.connection
+    };
+
+    runTimer.call(this);
+  };
+
+  runTimer = function() {
```

Comment:
Nitpicking: looks good because I know what it is, but in a few months this may not be this clear. Maybe adding a comment explaining that this is a token garbage collector might help?